workflow/rules/common.smk

```diff
@@ -417,8 +418,29 @@ for p in patient_list:
     if config.output["snv"] and config.output["visualization"]:
         visualizations.append(os.path.join(sdir, "visualization/index.html"))
 
+    # diversity measures
+    if not config.output["snv"] and config.output["diversity"]:
+        raise RuntimeError(
+            "Cannot generate diversity without calling variants (make sure to set `snv = True`) in config."
+        )
+
+    if config.output["snv"] and config.output["diversity"]:
+        diversity_measures.append(
+            os.path.join(sdir, "variants/SNVs/diversity_measures.csv")
+        )
+
```

Comment:
Is it necessary to include these files if we keep the aggregated one later on?

Reply:
No, we don't need to include these files. I removed them.